src/Microsoft.Identity.Web/TokenCacheProviders/Distributed/MsalDistributedTokenCacheAdapter.cs

```diff
@@ -114,9 +119,9 @@ protected override async Task<byte[]> ReadCacheBytesAsync(string cacheKey)
                         Size = result?.Length,
                     };
 
-                    _logger.LogDebug($"[MsIdWeb] Back propagate from Distributed to Memory, cache size: {result?.Length ?? 0}. ");
+                    Log.BackPropagateL2toL1(_logger, result?.Length ?? 0, null);
```

Comment:
Instead of `result?.Length ?? 0` we could use `memoryCacheEntryOptions.Size` as it's already computed